ntc_templates/templates/alcatel_aos_show_system.textfsm

```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
+Value CONTACT (.+,(?: https://\S+)?)
+Value NAME ([^,]+)
+Value LOCATION ([^,]+)
+Value Service (\d+)
```

Comment:
```suggestion
Value SERVICES (\d+)
```

Make plural to match raw output.
Capitalize capture group name.

Reply:
If this suggestion (and others) help, click the button here to accept them. Thank you!